projects/imagenet/experiments/base.py

```diff
@@ -38,7 +38,7 @@
 # See http://github.com/pytorch/examples/blob/master/imagenet/main.py
 DEFAULT = dict(
     # Results path
-    local_dir=os.path.expanduser("~/nta/results/imagenet"),
+    local_dir=os.path.expanduser("~/nta/results/experiments/imagenet"),
```

Comment:
Why the extra "experiments" folder?

Reply:
I am adding another layer, to keep some other stuff also under the results directory.